File: src/main/java/rak/planetGenerator/controllers/SliderControl.java
package rak.planetGenerator.controllers;

import javafx.fxml.FXML;
import javafx.scene.control.Label;
import javafx.scene.control.Slider;

public class SliderControl {
//Can we refactor common sliders into this?
	@FXML private Slider shadowFalloffSlider;
	
	@FXML private Label shadowFalloff;
	
	private float initialShadowStrength;
	
}

File: src/main/java/rak/planetGenerator/controllers/SliderControl.kt
package rak.planetGenerator.controllers

import javafx.fxml.FXML
import javafx.scene.control.Label
import javafx.scene.control.Slider

class SliderControl {
    //Can we refactor common sliders into this?
    @FXML private val shadowFalloffSlider: Slider? = null

    @FXML private val shadowFalloff: Label? = null

    private val initialShadowStrength: Float = 0.toFloat()

}
